### DISCUSSION TITLE
redshift data share

### DISCUSSION THREAD

**Discussion by misteliy** [dbeaver/dbeaver#16740] [Q&A]
#### System information: 
- Operating system (distribution) and version: MacOS 12.3.1
- DBeaver version: Version 22.0.4.202205011839
- Additional extensions

#### Connection specification:
- Database name and version: Redshift
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:

If I want to browse schemas and tables of a redshift data share I get following error:

<img width="567" alt="image" src="https://user-images.githubusercontent.com/893310/172558706-1b7eab68-3718-45bc-8c84-c81457e5a6a2.png">

#### Steps to reproduce, if exist:

Browsing any redshift data share.

**Comment by LonwoLonwo** [comment]
Hello @misteliy 

DBeaver supports Redshift data shares only in the Enterprise and Ulimate versions (Enterprise is recommended in your case).
You can try it for free with the trial license: https://dbeaver.com/trial/
